lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -265,7 +265,7 @@ class AutoEnrollmentViaCsv
         if student_result.is_general_error
           response_message =  student_result.response
         else
-          response_message = student_result.username + '  ('+ student_result.email + '):  ' + '   (' + student_result.response + ')'
+          response_message = student_result.username + '  (' + student_result.email + '):  ' + '   (' + student_result.response + ')'
```

Comment:
Is there are particular reason for using two spaces instead of one before/after parens and semicolons?